modules/exploits/multi/browser/chrome_object_create.rb

```diff
@@ -39,6 +44,11 @@ def initialize(info = {})
     register_advanced_options([
       OptBool.new('DEBUG_EXPLOIT', [false, "Show debug information during exploitation", false]),
     ])
+    deregister_options('DLL')
+  end
+
+  def library_path
+    File.join(Msf::Config.data_directory, "exploits", "CVE-2019-1458", "exploit.dll")
```

Comment:
I should probably add the sandbox escape as a target, so we can still use `--no-sandbox` payloads as target 0, and the sandbox escape (windows 7 only) as target 1.

Reply:
Some guy called timwr added a helper function for this.

```suggestion
    exploit_data('CVE-2019-1458', 'exploit.dll')
```

Reply:
library_path should be a path name, not raw binary data.